README.md

```diff
@@ -19,6 +19,11 @@ npm install //install dependencies
 npm start //start webpack-dev-server with cycle + hot reload
 ```
 
+## Testing
+```
+webpack --config webpack.test.config.js --watch
```

Comment:
since test-server is a custom named npm script(in contrast to test)
 it will have to be run via npm run test-server